samples/v1.2/Elements/BackgroundImage.Repeat.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Can these files follow similar naming convention for other element properties? E.g., `AdaptiveCard.BackgroundImage.Repeat.json`